config_examples/config_full.example.json

```diff
@@ -172,7 +172,23 @@
         "jwt_secret_key": "somethingrandom",
         "CORS_origins": [],
         "username": "freqtrader",
-        "password": "SuperSecurePassword"
+        "password": "SuperSecurePassword",
+        "ws_token": "a_secret_ws_token",
+        "enable_message_ws": false
+    },
+    "external_message_consumer": {
+        "enabled": false,
+        "producers": [
+          {
+            "name": "default",
+            "url": "ws://some.freqtrade.bot/api/v1/message/ws",
```

Comment:
they wouldn't anyway - as you'd assume 2 bots on one system (by using localhost) - which would require more work (diff. database, diff. ports, ...) - so i'm not sure one is better than the other (i'd change it to localhost too though - to avoid reaching out to some random IP that may resolve sometime in the future.

Reply:
:+1: